### PR TITLE
Adding About Section

### LINKED ISSUE #44: Add About Section
About section provides  clear and concise description of what the project is and also includes the project link .

### PR DESCRIPTION
**Pull request by aasthakourav20**
Fixes: #44 
Added About section that  clearly describes what the project is .

Closes: #44

### CODE REVIEW

**Comment by aasthakourav20** [comment]
Here is the about section !!
Kindly review the changes  &  add lables to it .





Screenshot (255)

**Comment by aasthakourav20** [comment]
Hey @dvjsharma , like this about section I would like to enhance other sections . Moreover I would like to add Services , Contact US , Blogs & FAQs int the footer . For the same I will raise an issue , if you want to add these sections kindly check the issues & assign it to me .

**Comment by aasthakourav20** [comment]
> Hey @dvjsharma , like this about section I would like to enhance other sections . Moreover I would like to add Services , Contact US , Blogs & FAQs int the footer . For the same I will raise an issue , if you want to add these sections kindly check the issues & assign it to me .

Hey @dvjsharma , any updates regarding this ?? Please reply & add labels to the PR

File: client/src/components/Navbar/navbar.jsx [hunk 1]
```diff
@@ -128,6 +128,13 @@ const Navbar = () => {
                     >
                         Home
                     </NavLink>
+                    <NavLink
+                        to="about"
+                        className="nav-link bg-white-500 hover:bg-white-600 active:bg-white-700 focus:outline-none focus:ring-white-300 hover:font-bold w-10 mr-8"
+                        activeClassName="nav-link-active underline"
+                    >
+                        About_Us 
+                    </NavLink>
```
Comment (dvjsharma) [inline client/src/components/Navbar/navbar.jsx:137]: I would suggest removing this form navbar. About us is generally present in footers only.

File: client/src/pages/About/index.jsx [hunk 2]
```diff
@@ -1,7 +1,33 @@
...
 const About = () => {
-    return <div>inside about</div>;
+    return <div className="bg-red-100"><br></br>
+          <div className=" rounded-lg p-4 mb-4">
+                <h1 className="text-6xl font-bold italic text-center">About Us</h1>
+            </div>
+            <div className=" rounded-lg p-4 mb-4">
+                <h3 className="text-2xl font-semibold text-center">Our Mission</h3><br></br>
+                <p className="text-gray-800 text-center">Drawn2Shoe is a user-friendly e-commerce platform, developed using ReactJS and Node.js. It offers customers the ability to purchase and personalize their shoes. The platform's intuitive interface ensures seamless navigation and usability across a variety of devices.</p>
+            </div>
```
Comment (dvjsharma) [inline client/src/pages/About/index.jsx:11]: Maybe try to keep this non-technical. User don't need to know which stack we are using, but will be happy to know what we are doing :)

File: client/src/pages/About/index.jsx [hunk 3]
```diff
@@ -1,7 +1,33 @@
...
+
+    Customer satisfaction is our top priority, and we value your feedback. Browse through our customer reviews to gain insights into the quality and craftsmanship of our products. Feel free to leave your own review to help fellow shoppers make informed decisions.
+
+    Don't forget to utilize our wishlist feature to save your favorite designs for future purchases. Whether you're planning your next custom creation or simply browsing for inspiration, your wishlist keeps your preferred styles organized and easily accessible.</p><br></br>
+                <p className="text-gray-800 text-center">Connect with Project Admin - Divij Sharma  </p>
+                <div className="flex mt-2 justify-center">
+                    <a className="mr-2 bg-gray-700 hover:bg-blue-500 text-white font-bold py-2 px-4 rounded text-center" href="https://www.linkedin.com/in/dvjsharma/">LinkedIn</a>
+                    <a className="bg-gray-700 hover:bg-gray-900 text-white font-bold py-2 px-4 rounded text-center" href="https://github.com/dvjsharma">GitHub</a>
```
Comment (dvjsharma) [inline client/src/pages/About/index.jsx:26]: Instead the admin section, put up the links we have in footer for social handles under get in touch heading. That would be more beneficial as a user POV.

**Review by dvjsharma — CHANGES_REQUESTED** [review]

**Comment by dvjsharma** [comment]
Hey @aasthakourav20. General design looks decent (background specifically). Yet, I would suggest modifying it so that we can stick to the theme. Maybe get some more inspiration and suggest screenshots. Example - https://dorik.com/blog/about-us-page-examples#4.-Cups

File: client/src/pages/About/index.jsx (hunk 2 above)
Comment (aasthakourav20) [reply client/src/pages/About/index.jsx:11]: > Maybe try to keep this non-technical. User don't need to know which stack we are using, but will be happy to know what we are doing :)

Yeah , I did it with general purpose just a simple demo . Surely I will enhance that part

**Comment by dvjsharma** [comment]
Any updates @aasthakourav20?

**Comment by dvjsharma** [comment]
Since there is no activity, I'm closing this PR. Please re-open if necessary.

**Pull request closed without merging**